src/quo2/components/switchers/utils.cljs

```diff
@@ -0,0 +1,35 @@
+(ns quo2.components.switchers.utils
```

Comment:
perhaps quo2.components.switchers.labels might be a more apt name? or does this file have more uses?